### ISSUE TITLE
pantry-2.0.0 is incompartible with persistent-template-2.8.0

### ISSUE THREAD

**Issue by buzden** [gentoo-haskell/gentoo-haskell#1019]
`persistent-template-2.8.0` (unlike 2.7.x) requires different set of language extensions [due to `DerivingStrategies` is now used](https://www.stackage.org/package/persistent-template#changes). Thus [some code](https://github.com/commercialhaskell/pantry/blob/master/src/Pantry/Storage.hs) (at least) in `pantry` does not compile.

**Comment by crocket** [comment]
```
src/Pantry/Storage.hs:115:1: error:
    Generating Persistent entities now requires the following language extensions:

DerivingStrategies
StandaloneDeriving

Please enable the extensions by copy/pasting these lines into the top of your file:

{-# LANGUAGE DerivingStrategies #-}
{-# LANGUAGE StandaloneDeriving #-}
```

**Comment by solpeth** [comment]
@buzden let me know if this still doesn't work for you. Should be fixed in 7a954b7.

**Comment by buzden** [comment]
It works well, thank you!
